e2e/typescript/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "noEmit": true,
+    "skipLibCheck": true,
+    "typeRoots": [],
+    "rootDir": "../../",
+    "target": "ES2020",
+    "lib": [
+        "ES2017",
+        "ES2020.BigInt",
+        "DOM"
+    ],
```

Comment:
I think the `libs` field can be removed. If not, then changed like this:

```suggestion
    "lib": [
        "ES2020",
        "DOM"
    ],
```

Reply:
It cannot be removed
Base tsconfig has `"lib": ["DOM"]` (i don't know why), so this errors occur: 
<img width="572" alt="image" src="https://github.com/DevExpress/DevExtreme/assets/84017191/670b9047-cbf9-4d64-9967-9cb5d9ca0b2d">

Reply:
changed to suggested

Reply:
thnx

Reply:
@VasilyStrelyaev, JFYI, same issue we've encountered previously